src/main/java/utils/FileUpload.java

```diff
@@ -0,0 +1,40 @@
+package utils;
+
+import java.awt.AWTException;
+import java.awt.Robot;
+import java.awt.Toolkit;
+import java.awt.datatransfer.Clipboard;
+import java.awt.datatransfer.StringSelection;
+import java.awt.event.KeyEvent;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+public class FileUpload {
+
+	public static void uploadImage(WebDriver driver,WebElement element,String path) {
+		Actions act =new Actions(driver);
+		act.moveToElement(element).click().perform();
+		
+		GetWaitMethod.getImplicitWait(driver);
+		
+		Robot robo;
+		try {
+			robo = new Robot();
+
+			StringSelection stringSelection = new StringSelection(path);
```

Comment:
26-28 can be moved a seprate method 

public Clipboard getClipboardItem(String item) {
  StringSelection stringSelection = new StringSelection(path);
  Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
  clipboard.setContents(stringSelection, null);
  return clipboard ;
}